src/CollectionTracker.tsx

```diff
@@ -156,17 +301,45 @@ export function CollectionTracker<P extends CollectionTrackerProps>({
         viewabilityConfig: _viewabilityConfig,
       }
 
-      // Wrap the rendered item with an action logger.
-      // TODO: Allow configuration so that controls within
-      // the rendered item can trigger different actions.
-      const _renderItem = ({ item }) => {
+      // Set up a state in which accessory view event handlers can
+      // set action type and name.
+      const [actionState, setActionState] = React.useState({
```

Comment:
Feel free to ignore this comment.

I've previously had action instrumentation logic that accidentally started multiple actions.  E.g. the top-level handler sets an action but a common method is used that also starts an action.  This current implementation doesn't currently catch this.

Not sure if this important.  This can also be changed later.  I could see failing in dev mode or supporting a list of them.

Reply:
Similar to the above case, we can eventually add warnings if you call `logAction` from inside the scope of an action that CollectionTracker should handle automatically.